src/tools/cgxset.c

```diff
@@ -289,6 +289,11 @@ int main(int argc, char *argv[])
 			goto err;
```

Comment:
@drakenclimber good catch.  One improvement would be to replace the goto `err`  with goto `cgroup_free_err`.   The `cgroup` is allocated and needs to free'd too.